ts/features/bonus/bpd/analytics/index.ts

```diff
@@ -133,6 +140,53 @@ const trackAction = (mp: NonNullable<typeof mixpanel>) => (
             ? action.payload.kind
             : action.payload.value.message
       });
+
+    // Amount
+    case getType(bpdAmountLoad.request):
+      return mp.track(action.type, { reason: action.payload });
+    case getType(bpdAmountLoad.success):
+      return mp.track(action.type);
```

Comment:
```suggestion
    case getType(bpdAmountLoad.success):
      return mp.track(action.type, {
        awardPeriodId: action.payload.awardPeriodId
      });
```
My fault, I forgot this case!